src/main/java/ch/njol/skript/Skript.java

```diff
@@ -657,7 +656,11 @@ protected void afterErrors() {
 					// Ignore late init (scripts, etc.) in test mode
 					Bukkit.getScheduler().runTaskLater(Skript.this, () -> {
 						// Delay is in Minecraft ticks.
-						long shutdownDelay = 0;
+						AtomicLong shutdownDelay = new AtomicLong(), milliseconds = new AtomicLong(),
+							tests = new AtomicLong(), fails = new AtomicLong(),
+							ignored = new AtomicLong(), size = new AtomicLong();
```

Comment:
All of these AtomicLong's are now being initialized as -1L rather than the existing 0L
```suggestion
						AtomicLong shutdownDelay = new AtomicLong(0), milliseconds = new AtomicLong(0),
							tests = new AtomicLong(0), fails = new AtomicLong(0),
							ignored = new AtomicLong(0), size = new AtomicLong(0);
```

Reply:
The javadoc for the constructor says the initial value is 0, I'll add it for making it explicit though